.github/workflows/get-cloud-api-spec.yml

```diff
@@ -0,0 +1,65 @@
+name: Fetch and Save Cloud API Spec
+
+on:
+  workflow_dispatch: # Allows manual trigger of the workflow
+  repository_dispatch: # Allows other repositories to trigger this workflow
+    types: [trigger-cloud-api-docs]
+
+jobs:
+  fetch-and-save-cloud-api-spec:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository (main branch)
+        uses: actions/checkout@v3
+        with:
+          ref: 'main'
+          path: redpanda-docs
+          token: ${{ secrets.ACTIONS_BOT_TOKEN }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+
+      - name: Install dependencies
+        run: |
+          cd ./redpanda-docs/scripts/get-api-specs
+          npm install
+
+      - name: Create directories if they do not exist
+        run: mkdir -p ./redpanda-docs/modules/ROOT/attachments
+
+      - name: Run script and save output
+        run: node ./redpanda-docs/scripts/get-api-specs/get-cloud-api-spec.js > ./redpanda-docs/modules/ROOT/attachments/cloud-api.yaml
```

Comment:
can this fail if the fail already exists?

Reply:
I've updated it to save at the root of the repo. It's just a temporary file that we move to the `attachments/` directory after checking out the `api` branch.